Tools/TestWebKitAPI/ios/DragAndDropSimulatorIOS.mm

```diff
@@ -50,7 +50,6 @@
 using namespace TestWebKitAPI;
 
 @interface DragAndDropSimulator () <UIDragAnimating>
```

Comment:
Can't we toss the whole category now? Or does something still use the _dragAndDropSimulator?

Reply:
I suppose we could technically remove the declaration of `-_setNeedsDropPreviewUpdate:` since it's just used as an internal method now. We do still implement `UIDragAnimating` methods, though.

Reply:
I.... use it?
[[instance _dragAndDropSimulator] _setNeedsDropPreviewUpdate:instance];
further down in the file.